### PR TITLE
Fix and clean up anti-aliased line drawing

### PR DESCRIPTION
**Pull request by stefanv**

### CODE REVIEW

File: skimage/draw/_draw.pyx [hunk 1]
```diff
@@ -37,14 +37,14 @@ def _coords_inside_image(rr, cc, shape, val=None):
     return rr[mask], cc[mask]
 
 
-def line(Py_ssize_t y, Py_ssize_t x, Py_ssize_t y2, Py_ssize_t x2):
```
Comment (sciunto) [inline skimage/draw/_draw.pyx]: Why not x0,y0 instead of x,y? to be consistent with line_aa...

Comment (stefanv) [inline skimage/draw/_draw.pyx]: That looks like a bug that slipped through while renaming variables.

**Comment by ahojnnes** [comment]
Any reason why not change to x->c and y->r as well?

**Comment by stefanv** [comment]
No, I just didn't want to do that as part of this PR

**Comment by stefanv** [comment]
(the whole file needs to be checked)

**Comment by JDWarner** [comment]
Merging with only AppVeyor input as Travis is backed up. Closes #1563.

**Pull request merged**